dkc/core/models/user_quota.py

```diff
@@ -1,26 +1,29 @@
 from typing import Type
 
+from django.apps import apps
 from django.contrib.auth.models import User
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django_girders.models import SelectRelatedManager
 
 
-class Quota(models.Model):
-    allocation = models.BigIntegerField(default=0)
-    user = models.OneToOneField(User, null=True, on_delete=models.CASCADE)
-    # root_set
+class UserQuota(models.Model):
+    user = models.OneToOneField(
+        User, primary_key=True, on_delete=models.CASCADE, related_name='quota'
+    )
+    used = models.PositiveBigIntegerField(default=0)
+    allowed = models.PositiveBigIntegerField()
```

Comment:
The default value can be a callable. We should define a local private function, and use it to return the setting value.

Reply:
Note, it can't be a lambda, since that can't be serialized to a migration.